katsdpimager/test/test_primary_beam.py

```diff
@@ -68,7 +68,7 @@ def test_given_frequency(self) -> None:
         np.testing.assert_allclose(beam[0, 0, 0, N // 2, N // 2], 1.0, rtol=1e-2)
         # If it fails due to code/model changes, uncomment to inspect visually:
         # self._show_beam(beam)
-        assert_equal(hashlib.md5(beam[0, 0, 0]).hexdigest(), '8c3ff23e1b8fb76e8b3efd5a88cbf632')
+        assert_equal(hashlib.md5(beam[0, 0, 0]).hexdigest(), '30726bc4a42ddb1813dcdf3a1d54251c')
```

Comment:
Max difference was around 2e-8 (abs), which is actually pretty good given that it's now being computed in single precision.